pydm/main_window.py

```diff
@@ -498,3 +501,59 @@ def _confirm_quit(self, callback):
     def show_macro_window(self):
         macro_window = MacroWindow(self)
         macro_window.show()
+
+    def add_menu_items(self):
+        # create the custom menu with user given items 
+        items = self.display_widget().menu_items()
+        if len(items) == 0:
+            self.ui.menuCustomActions.setEnabled(False)
+        else:
+            self.ui.menuCustomActions.setEnabled(True)
```

Comment:
This makes sense to me. I changed the File menu options to not appear if not set, and instead of greying out the Actions menubar, it is simply empty, which seems to me representative of the state of the display (i.e., this is an available section that works, but is unfilled).